templates/default/tests/markdownFeatures.spec.js

```diff
@@ -0,0 +1,136 @@
+const { test, expect } = require('@playwright/test');
+const { MarkdownPage } = require('./markdown-page');
+
+test.describe('wiki links', () => {
+  test.beforeEach(async ({ page }) => {
+    const Page = new MarkdownPage(page);
+    await Page.goto('/test/fixtures/markdownFeatures');
+    await Page.getData(page);
+  });
+
+  test('parses a wiki link', async ({ page }) => {
+    const link = page.locator('#wikiLink > p > a');
+    await expect(link).toHaveClass('internal');
+    await link.click();
+    await expect(page).toHaveURL('/docs/roadmap');
```

Comment:
This test will fail if we move the roadmap page to some other location or we just delete it. Tests should be totally decoupled from the actual content so that they don't rely on things that may easily change in the future. => Let's create a fixture page for this purpose and link to it in `markdownFeatures.md`.